core/utils/poly-look/src/css/themes.css

```diff
@@ -0,0 +1,21 @@
+:root,
+.poly-theme {
+  --theme-font-base: var(--jost);
+  --theme-font-medium: var(--jost-medium);
+  color: var(--theme-secondary-color);
```

Comment:
Wait, the main theme has a "secondary" color? Maybe call it "fallback" color?

Reply:
Color is font-color and i want the font to be the secondary color here (contrast to background)

Reply:
I get that, the problem again is the name. If you want it to be contrast color, maybe name it that way. The thing is that the meaning of "secondary" is not at all clear in the context (or its coincidence with other cases). What happens if you change the value? Will you need to change also the variable assignment here? Or just use whatever color is defined as "secondary"?

Reply:
But there will be multiple contrasting ones making it unclear which is meant by that. That's the point when primary color is the background then the texts would contrast it with the secondary color.

Reply:
I totally get your point don't get me wrong here. I see the problem with primary/ secondary being vague names with no meaning tied to it. At the same time I feel like background/ contrast would tie them too much to specific things, which would feel even weirder using it. Giving a button a font-color of background color? weird

Reply:
You're right. Let's leave it for another occasion then. Thanks for understanding it :-)